.github/workflows/storybook.yml

```diff
@@ -8,9 +13,10 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: "16"
+          node-version: "18"
           cache: "npm"
+
       - name: install dependencies
         run: npm ci
-      # - name: run build
-      #   run: npm run build-storybook
+      - name: run build
+        run: npm run build-storybook
```

Comment:
Could both Actions be merged into one and have the deploy with an `if` condition for the branch `main`?

Reply:
It is not a blocker for this PR, it was just an idea. For example

Action
- build
- deploy with the `if` condition, I think something like this `if: github.ref == 'refs/heads/master'`

Reply:
I got the actions combined now.

And the ifs work correctly